foxglove_data_platform/client.py

```diff
@@ -203,79 +199,58 @@ def delete_event(
         event_id: The id of the event to delete.
         """
         response = requests.delete(
-            self.__url__(f"/beta/device-events/{event_id}"),
+            self.__url__(f"/v1/events/{event_id}"),
             headers=self.__headers,
         )
-        json_or_raise(response)
+        return json_or_raise(response)
 
     def get_events(
         self,
-        device_id: Optional[str] = None,
-        device_name: Optional[str] = None,
+        *,
+        device_id: str,
```

Comment:
Was making `device_id` required intentional? It's still optional in the API.

Reply:
Yup, i mistook it for being required since the previous API had *either* device ID or device name required, and device name was removed.